deepfence_server/reporters/notification/scan.go

```diff
@@ -0,0 +1,155 @@
+package notification
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/deepfence/ThreatMapper/deepfence_server/model"
+	"github.com/deepfence/ThreatMapper/deepfence_utils/directory"
+	"github.com/deepfence/ThreatMapper/deepfence_utils/log"
+	"github.com/neo4j/neo4j-go-driver/v5/neo4j"
+)
+
+func GetScans(ctx context.Context, scanTypes []string, statues []string) (model.NotificationGetScanResponse, error) {
+	response := model.NotificationGetScanResponse{}
+	var err error
+	for _, scanType := range scanTypes {
+		switch scanType {
+		case "vulnerability":
```

Comment:
I think this can be simplified by having a map like this https://github.com/deepfence/ThreatMapper/blob/main/deepfence_utils/utils/constants.go#L137

Reply:
@ibreakthecloud Can you also check the lint errors?

Reply:
only possible changes where using constants, we cannot simplify this since each scan result has to go in different fields of struct. 
I did the changes, please check again